app/src/main/java/org/thoughtcrime/securesms/conversation/v2/input_bar/InputBarRecordingView.kt

```diff
@@ -45,6 +45,14 @@ class InputBarRecordingView : RelativeLayout {
     val recordButtonOverlay: RelativeLayout
         get() = binding.recordButtonOverlay
 
+    // Static constants for animation durations in milliseconds
+    companion object {
+        val AnimateLockDurationMS = 250L
+        val DotAnimationDurationMS = 500L
+        val DotPulseAnimationDurationMS = 1000L
+        val ShowHideDurationMS = 250L
```

Comment:
I think we can use const on the constants:

```suggestion
        const val AnimateLockDurationMS = 250L
        const val DotAnimationDurationMS = 500L
        const val DotPulseAnimationDurationMS = 1000L
        const val ShowHideDurationMS = 250L
```

Do people have opinion on the casing of constants? Like `ANIMATE_LOCK_DURSTION_MS`?

Reply:
Yeah they usually are all caps. We can start making it a consistent decision

Reply:
Done.

```
const val ANIMATE_LOCK_DURATION_MS        = 250L
const val DOT_ANIMATION_DURATION_MS       = 500L
const val DOT_PULSE_ANIMATION_DURATION_MS = 1000L
const val SHOW_HIDE_VOICE_UI_DURATION_MS  = 250L
```